frontend/src/components/Charts/DepthChart/index.tsx

```diff
@@ -0,0 +1,346 @@
+                  <Grid item xs={12}>
+                    <PaymentText 
+                      othersText={t("Others")} 
+                      verbose={true} 
+                      size={20} 
+                      text={order.payment_method}
+                    />
+                  </Grid>
+                </Grid>
+              </Box>
+            </Grid>
+          </Grid>
+        </Grid>
+      </Paper>
+    ) : <></>
+  }
+
+  const formatAxisX = (value: number): string => {
+    if (xType === 'base_amount') {
+      return value.toString()
+    }
+    return `${value}%`
+  }
+  const formatAxisY = (value: number): string => `${value}BTC`
+
+  const rangeSteps = xType === 'base_amount' ? 200 : 0.5
+
+  const handleOnClick: PointMouseHandler = (point: Point) => {
+    history.push('/order/' + point.data?.order?.id);
+  }
+  
+  return bookLoading || !center || enrichedOrders.length < 1 ? (
+    <div style={{display: "flex", justifyContent: "center", paddingTop: 200, height: 420 }}>
+      <CircularProgress />
+    </div>
+  ) : (
+    <Grid container style={{ paddingTop: 15 }}>
+      <Grid 
+        container 
+        direction="row"
+        justifyContent="space-around"
+        alignItems="flex-start"
+        style={{ position: "absolute" }}
+      >
+        <Grid container justifyContent="flex-start" alignItems="flex-start" style={{ paddingLeft: 20 }}>
+          <Select
+            value={xType}
+            onChange={(e) => setXType(e.target.value)}
+          > 
+            <MenuItem value={"premium"}>
+              <div style={{display:'flex',alignItems:'center', flexWrap:'wrap'}}>
+                {t("Premium")}
+              </div>
+            </MenuItem>
+            <MenuItem value={"base_amount"}>
+              <div style={{display:'flex',alignItems:'center', flexWrap:'wrap'}}>
+                {t("Price")}
+              </div>
+            </MenuItem>
+          </Select>
+        </Grid>
+      </Grid>
+      <Grid 
+        container 
+        direction="row"
+        justifyContent="center"
+        alignItems="center"
+      >
+        <Grid container justifyContent="center" alignItems="center">
+          <Grid item>
+            <IconButton onClick={() => setXRange(xRange + rangeSteps)}>
+              <RemoveCircleOutline />
+            </IconButton>
+          </Grid>
+          <Grid item>
+            <Box justifyContent="center">
+              {xType === 'base_amount' ? `${center} ${currencyDict[currencyCode]}` : `${center}%`}
+            </Box>
+          </Grid>
+          <Grid item>
+            <IconButton onClick={() => setXRange(xRange - rangeSteps)} disabled={xRange <= 1}>
+              <AddCircleOutline />
+            </IconButton>
+          </Grid>
+        </Grid>
+      </Grid>
+      <Grid container style={{ height: 357, padding: 15 }}>
+        <ResponsiveLine 
+          data={series} 
+          enableArea={true}
+          useMesh={true}
+          animate={false}
```

Comment:
In general, the animation feature is cool, but on this case the X axis scale changes so drastically (from premium to price) that it really looks odd.